### DISCUSSION TITLE
Problem for login Notion?

### DISCUSSION THREAD

**Discussion by Amppa** [webclipper/web-clipper#1039] [Q&A]
Hi guys,
I can't connect with Notion.so. 
Chrome addon keeps showing:
`Cannot read properties of undefined (reading 'ceebd4fc...`

And firefox addon showing:
`M is undefined`

What should I do next?

**Comment by LoimyZh** [comment]
I also have the same problem, both on Chrome and Edge keep showing:
`Cannot read properties of undefined (reading '4f861ec6-f84d-43a3-80d9-4696b23ccf97')`
Did you solve that?

**Comment by DiamondYuan** [comment]
try new version
